.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ env:
   REGISTRY: ghcr.io
   SEGMENT_DOWNLOAD_TIMEOUT_MINS: 2
   SQLX_OFFLINE: true
-  RUSTC_VERSION: 1.73.0
+  RUSTC_VERSION: 1.75.0
```

Comment:
we should also update the dockerfile rust version

Reply:
Ah, I thought I got all of them. Pushed up another commit that fixes this and another one as well.